app/styles/app.css

```diff
@@ -248,17 +248,38 @@ input {
 
 .searchableMetricSelector .searchBar {
     border-bottom: 1px solid #eee;
+    position: sticky;
+    top: 0;
+    background-color: white;
+}
+
+.searchableMetricSelector .metricGroup {
+    font-weight: bold;
 }
 
-.searchableMetricSelector .searchResults p {
+.searchableMetricSelector .metricResult {
     border-bottom: 1px solid #D3D3D3;
 }
 
-.searchableMetricSelector .searchResults p:hover {
+.searchableMetricSelector .metricResult:hover{
+    background-color: rgba(180, 180, 180, 0.5);
```

Comment:
You can make a CSS variable (`var`) for this. Cloud9 complains, but look at the synntnax used in other places in the file, that's valid.

Reply:
@fabrego524 did you see this comment?